charts/openshift-metering/templates/presto/hive-metastore-statefulset.yaml

```diff
@@ -120,6 +120,14 @@ spec:
 {{- end }}
         resources:
 {{ toYaml .Values.presto.spec.hive.metastore.resources | indent 10 }}
+      - name: ghostunnel-server
+        command: ["ghostunnel"]
+        args: ["server", "--listen", "localhost:8443", "--target", "localhost:9083", "--key", "/opt/hive/tls/tls.key", "--cert", "/opt/hive/tls/tls.crt", "--cacert", "/opt/hive/tls/ca.crt", "--allow-cn", "Any"]
+        image: quay.io/openshift/origin-ghostunnel
+        volumeMounts:
+        - name: hive-metastore-secret
```

Comment:
Probably rename the volume and volumeMount names to "hive-tls-secrets" to better represent what the volume/mounts are.

Reply:
Can do, but hive-server will have tls secrets as well. What will the hive server secret name be to keep them distinct?

Reply:
`hive-metastore-tls-secrets`?